object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -57,7 +57,7 @@
 public class ObjectConstructionAnnotatedTypeFactory extends BaseAnnotatedTypeFactory {
 
   /** The top annotation. Package private to permit access from the Transfer class. */
-  final AnnotationMirror TOP;
+  public final AnnotationMirror TOP;
```

Comment:
@kelloggm we now need to access this top type from a different package.  We could change this public field to an accessor method.  But is there a cleaner way in general to access the top type of a type system?

Reply:
If you can access the [QualifierHierarchy](https://checkerframework.org/api/org/checkerframework/framework/type/QualifierHierarchy.html), it has methods to get the top annotation.

Reply:
For an annotation without arguments like `@CalledMethodsTop`, though, I think exposing it as a public final field on the AnnotatedTypeFactory is fine. Some of the internal Checker Framework checkers do that, for instance: https://github.com/typetools/checker-framework/blob/master/checker/src/main/java/org/checkerframework/checker/index/lowerbound/LowerBoundAnnotatedTypeFactory.java#L100

Reply:
I agree that is a more direct way.